src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,17 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        int b;
+        String bianry = "";
```

Comment:
let's not forget about checklist before submitting solution🙂
https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html

Reply:
i have fixed.